templates/tools/dockerfile/test/cxx_jessie_x64/Dockerfile.template

```diff
@@ -29,4 +29,4 @@
   RUN apt-get update && apt-get -y install gcc-4.8 gcc-4.8-multilib g++-4.8 g++-4.8-multilib && apt-get clean
 
   # Define the default command.
```

Comment:
can we remove <%include file="../../clang_update.include"/>  from this file?

Reply:
Done